src/arcaflow_plugin_sdk/test_schema.py

```diff
@@ -531,26 +555,25 @@ def test_unserialize(self):
             s_type.unserialize([])
         # Mismatching key value
         with self.assertRaises(ConstraintException):
-            s_type.unserialize({
-                self.discriminator_default: "a", "b": "Hello world!"})
+            s_type.unserialize(
+                {discriminator_default: "a", "b": "Hello world!"}
+            )
```

Comment:
"discriminator_default" reads kind of funny, here (and my attempt to describe why reads even funnier, so I've redacted it).  The point is, wouldn't it sound better in this usage if it were `default_discriminator`?

Reply:
yeah it reads funny, but it's also shorter than `discriminator_field_name_default`

Reply:
renamed